src/Xamarin.Android.Build.Tasks/Resources/ApplicationRegistration.java

```diff
@@ -1,4 +1,4 @@
-﻿package mono.android.app;
+﻿package JAVA_PACKAGE_NAME;
```

Comment:
Why change the package name? Why not just always leave it as `mono.android.app`?

Reply:
or always have the new name, and fix the .NET for Android build.  Regardless, I don't see a reason to have a *variable* package name here.